packages/nuxt-locokit/src/runtime/tailwind.config.ts

```diff
@@ -25,8 +21,26 @@ export default {
         'secondary-light': 'var(--color-secondary-light)',
         'secondary-dark': 'var(--color-secondary-dark)',
         error: colors.red['500'],
+        // 'oyster-bay': 'var(--color-oyster-bay)',
+        theme: {
+          DEFAULT: 'var(--theme-bg-color)',
+          hover: 'var(--theme-bg-color-hover)',
+          text: 'var(--theme-text-color)',
+        },
+      },
+      screens: {
+        sm: '640px',
+        // => @media (min-width: 640px) { ... }
+        md: '768px',
+        // => @media (min-width: 768px) { ... }
+        lg: '1024px',
+        // => @media (min-width: 1024px) { ... }
+        xl: '1280px',
+        // => @media (min-width: 1280px) { ... }
+        '2xl': '1536px',
+        // => @media (min-width: 1536px) { ... }
```

Comment:
I don't understand why this is necessary.
I think these are the default values : https://tailwindcss.com/docs/responsive-design

Reply:
To explicit the default values in the project

Reply:
Well, as there are already defined in the package, I think this is not useful ?